editor/editor_file_dialog.cpp

```diff
@@ -696,7 +698,7 @@ void EditorFileDialog::_item_menu_id_pressed(int p_option) {
 					path = path.get_base_dir();
 				}
 			}
-			OS::get_singleton()->shell_open(String("file://") + path);
+			OS::get_singleton()->shell_show_in_explorer(path, open_folder);
```

Comment:
Note that in case a directory is selected the menu entry is "Open in File Manager", it's expected to open the folder instead of just showing it:
https://github.com/godotengine/godot/blob/0e7c98061856a171838f4ae41e2ca925158e24aa/editor/editor_file_dialog.cpp#L626

I don't think it needs to be changed.

```suggestion
			OS::get_singleton()->shell_show_in_explorer(path, true);
```
(and remove `open_folder`)

Reply:
I think it is needed.

If user want to copy a resource folder in file manager, if alaway open folder, they must select target folder's up level folder and show it in explorer, then find the target folder again.

Additionally, I think let behavior of show folder same as show file is more intuitive.

Specially, if click at background, user want to open current folder obviously.

Reply:
Hmm, I guess sometimes you want to open the right clicked folder and sometimes you want to select it. Maybe in case of right clicking a directory both menu entries ("Open in File Manager" and "Show in File Manager") could be shown? :thinking:

cc @akien-mga Any opinion on this?